modules/cli/pages/cli-intro.adoc

```diff
@@ -39,46 +39,117 @@ You can collect diagnostics through the command-line interface by using the xref
 [#server-tools-packages]
 == Server Tools Packages
 
-For convenience, Couchbase provides a  tools package  that contains the following utilities:
+For convenience, Couchbase provides Server developer tools package and Server admin tools package.
+
+=== Server Developer Tools Packages
+
+Server developer tools package contains the following utilities:
+
+* xref:tools:cbimport.adoc[`cbimport`]
+* xref:tools:cbexport.adoc[`cbexport`]
+* xref:cli:cbq-tool.adoc[`cbq`]
+* xref:backup-restore:cbbackupmgr.adoc[`cbbackupmgr`]
+* xref:cbcli:couchbase-cli.adoc[`couchbase-cli`]
+* cbstats
+* mctimings
+* mcstat
+
+This package lets you install the following Server developer tools on systems where you have not installed Couchbase Server.
+
+Download the command line tools package for your platform from the following links:
+
+* Linux: https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-dev-tools_{tools-ver}-linux_x86_64.tar.gz[]
+* Linux aarch64: https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-dev-tools_{tools-ver}-linux_aarch64.tar.gz[]
+* macOS: https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-dev-tools_{tools-ver}-macos_x86_64.zip[]
+* macOS arm64: https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-dev-tools_{tools-ver}-macos_arm64.zip[]
+* Windows: https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-dev-tools_{tools-ver}-windows_amd64.zip[]
+
+Unzip or untar the packages, and the binaries are ready to run.
+For example:
+
+[source,console,subs="attributes+"]
+----
+wget https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-dev-tools_{tools-ver}-linux_x86_64.tar.gz
+
+tar -xf couchbase-server-dev-tools_{tools-ver}-linux_x86_64.tar.gz
+----
+
+Each package also contains a `README` file and a copy of the software license.
+
+NOTE: On Windows, you must have a recent version of the Microsoft Visual {cpp} Redistributable runtime libraries installed.
+If you do not have these libraries installed, download them from https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170[Microsoft Visual {cpp} Redistributable latest supported downloads].
+
+=== Server Admin Tools Packages
+
+Server admin tools package contains the following utilities:
 
 * xref:tools:cbimport.adoc[`cbimport`]
 * xref:tools:cbexport.adoc[`cbexport`]
 * xref:cli:cbq-tool.adoc[`cbq`]
 * xref:backup-restore:cbbackupmgr.adoc[`cbbackupmgr`]
+* couchbase-cli
+* cbstats
+* cbdatarecovery
+* mctimings
+* mcstat
+* mctestauth
 
-This package lets you install these tools on systems where you have not installed Couchbase Server.
+This package lets you install the following Server admin tools on systems where you have not installed Couchbase Server.
```

Comment:
Also here, it says "the following Server admin tools", but the list is above, so just remove the word "following" or move the sentence to before the list.